modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/components/SpaceSticker.tsx

```diff
@@ -6,44 +6,50 @@
 import ClaySticker from '@clayui/sticker';
 import React from 'react';
 
-function getDisplayType(
-	char: string
-): React.ComponentProps<typeof ClaySticker>['displayType'] {
-	const validDisplayTypes: React.ComponentProps<
-		typeof ClaySticker
-	>['displayType'][] = [
-		'outline-0',
-		'outline-1',
-		'outline-2',
-		'outline-3',
-		'outline-4',
-		'outline-5',
-		'outline-6',
-		'outline-7',
-		'outline-8',
-		'outline-9',
-	];
+export type LogoColor = React.ComponentProps<typeof ClaySticker>['displayType'];
 
-	return validDisplayTypes[char.charCodeAt(0) % validDisplayTypes.length];
+export const logoColors: LogoColor[] = [
+	'outline-0',
+	'outline-1',
+	'outline-2',
+	'outline-3',
+	'outline-4',
+	'outline-5',
+	'outline-6',
+	'outline-7',
+	'outline-8',
+	'outline-9',
+];
+
+function getDisplayType(char: string): LogoColor {
+	return logoColors[char.charCodeAt(0) % logoColors.length];
 }
 
 export default function SpaceSticker({
+	className,
 	displayType,
+	hiddenName,
```

Comment:
Hi @albertininm following the same review that @adolfopa did [here](https://github.com/liferay-content-management/liferay-portal/pull/6679#discussion_r2111967393) to the `hiddenName` parameter:

> Can this be named as an action instead of an adjective?

I would use `hideName` or `showName`. I suppose the former is better because we don't need to have a default value.

Reply:
Otherwise it seems that the component hast 2 attributes: a `name` and a `hiddenName`.

Reply:
thanks @marcogalluzzi that makes sense!!!